frontend/nuxt.config.ts

```diff
@@ -191,10 +189,11 @@ export default defineNuxtConfig({
   },
   plausible: {
     enabled: !isTest,
-    logIgnoredEvents: !isProductionBuild,
-    trackLocalhost: !isProdNotPlaywright,
-    autoPageviews: isProdNotPlaywright,
-    domain: import.meta.env.SITE_DOMAIN,
-    apiHost: import.meta.env.PLAUSIBLE_SITE_URL,
+    ignoredHostnames: ["localhost", "staging.openverse.org"],
+    // Set `NUXT_PUBLIC_PLAUSIBLE_LOG_IGNORED_EVENTS` to "true" to log in the dev environment.
+    logIgnoredEvents: false,
```

Comment:
Any reason not to log by default? Since these logs are always on the client it doesn't seem too intrusive or disruptive. And in production since the events are not ignored, they wouldn't log. Is that right?

Reply:
If set to true, it will log in two cases:
- the plausible domain is on the list of ignored domains (so, it will log in staging, and in dev mode)
- the user has switched off analytics (this might confuse some users if they open console and see events logged after they've turned off the analytics)

I think that we should only be logging in the dev mode; not on staging or when the user has switched off analytics.

After having more thought about that, I think the best way to fix this would be to set `logIgnoredEvents` to `true` in `nuxt.config`, but add the `NUXT_PUBLIC_PLAUSIBLE_LOG_IGNORED_EVENTS=false` to staging and production. What do you think, @zackkrida?

Reply:
I added this variable to https://github.com/WordPress/openverse-infrastructure/pull/997

Reply:
@obulat thank you for the extra info, I didn't consider the case of users who have disabled analytics! 

The new approach sounds good to me with one small exception: why wouldn't we want log the events on staging?

Reply:
Since the staging is partly public-facing, I thought it would be better to disable logging there. But disabling it on staging would not allow us to understand if Plausible events are being fired. I'll update this change.